### ISSUE TITLE
Get actor from token

### ISSUE THREAD

**Issue by poksiala** [radiodiodi/sahkopiikki#4]
If I understood changes in https://github.com/radiodiodi/sahkopiikki/pull/2 correctly actor of transaction is defined in frontend code and backend does not verify this leaving it open for abuse. Actor should be acquired from the request auth instead.

**Comment by jantuomi** [comment]
Yeah that's right. That was first attempted but proved to be difficult for a couple of reasons.

A totally valid issue, but not super important in a system built around trust.
